apps/extension/src/ui/apps/popup/pages/AddEthereumNetwork.tsx

```diff
@@ -1,44 +1,200 @@
 import { KnownRequestIdOnly } from "@core/libs/requests/types"
+import { log } from "@core/log"
 import { AppPill } from "@talisman/components/AppPill"
 import { notify } from "@talisman/components/Notifications"
-import { GlobeIcon } from "@talismn/icons"
+import { EvmNetwork } from "@talismn/chaindata-provider"
+import { GlobeIcon, InfoIcon } from "@talismn/icons"
+import { classNames } from "@talismn/util"
 import { api } from "@ui/api"
 import { balancesHydrateAtom } from "@ui/atoms"
-import { NetworksDetailsButton } from "@ui/domains/Ethereum/NetworkDetailsButton"
+import { ChainLogo } from "@ui/domains/Asset/ChainLogo"
+import { NetworkDetailsButton, NetworkDetailsLink } from "@ui/domains/Ethereum/NetworkDetailsButton"
+import { useEvmNetwork } from "@ui/hooks/useEvmNetwork"
 import { useRequest } from "@ui/hooks/useRequest"
-import { useAtomValue } from "jotai"
-import { useCallback } from "react"
+import { requestsAtom } from "@ui/hooks/useRequests"
+import useToken from "@ui/hooks/useToken"
+import { atom, useAtomValue } from "jotai"
+import { ChangeEventHandler, FC, ReactNode, useCallback, useMemo, useState } from "react"
 import { Trans, useTranslation } from "react-i18next"
 import { useParams } from "react-router-dom"
-import { Button } from "talisman-ui"
+import { Button, Tooltip, TooltipContent, TooltipTrigger } from "talisman-ui"
+import { AddEthereumChainParameter, isHex, toHex } from "viem"
 
 import { PopupContent, PopupFooter, PopupHeader, PopupLayout } from "../Layout/PopupLayout"
 
+// TODO once finalized with Diogo, convert as style.css classes, then move to talisman-ui along with a test page in playground
+const Radio: FC<{
```

Comment:
Looks good, can convert my selector in the QR account configure component to follow